### PR TITLE
Matrix Room Manifestation

### LINKED ISSUE #564: Mention Matrix manifestation
Now that the Matrix is being used the add-ons room should be mentioned in the "Questions about add-on development" section on https://extensionworkshop.com/community/ .

https://chat.mozilla.org/#/room/#addons:mozilla.org

### PR DESCRIPTION
**Pull request by grlwholifts**
Fixes #564 

- Added matrix rooms [ #add-ons, #addons-reviewers ] under `Questions about add-on development` section on https://extensionworkshop.com/community/ 

cc: @bjherbison

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -63,9 +63,9 @@ Subscribe to our [mailing list](https://mail.mozilla.org/listinfo/dev-addons) de
 
 ### Questions about add-on development
 
-You can get help and support by posting to [Mozilla's community forum](https://discourse.mozilla.org/c/add-ons), [Matrix Chat Rooms](https://wiki.mozilla.org/Matrix): [#add-ons](https://matrix.to/#/#add-ons:mozilla.org), [#addons-reviewers](https://matrix.to/#/#addon-reviewers:mozilla.org), [Stack Overflow](http://stackoverflow.com/questions/tagged/firefox-addon), or the [developer mailing list](https://mail.mozilla.org/listinfo/dev-addons).
+You can get help and support by posting to [Mozilla's community forum](https://discourse.mozilla.org/c/add-ons), [Matrix Chat Rooms](https://wiki.mozilla.org/Matrix): [#add-ons](https://bit.ly/2PKCHuH), [#addons-reviewers](https://bit.ly/32Odp4i), [Stack Overflow](http://stackoverflow.com/questions/tagged/firefox-addon), or the [developer mailing list](https://mail.mozilla.org/listinfo/dev-addons).
```
Comment (championshuttler) [inline community.md]:
```suggestion
You can get help and support by posting to [Mozilla's community forum](https://discourse.mozilla.org/c/add-ons), [Matrix Chat Rooms](https://wiki.mozilla.org/Matrix): [#add-ons](https://mzl.la/2u8ZGbg), [#addons-reviewers](https://bit.ly/32Odp4i), [Stack Overflow](http://stackoverflow.com/questions/tagged/firefox-addon), or the [developer mailing list](https://mail.mozilla.org/listinfo/dev-addons).
```

Comment (caitmuenster) [reply community.md]: Let's break this up a little bit. It should read: 

You can get help and support by posting to [Mozilla's community forum](https://discourse.mozilla.org/c/add-ons), [Stack Overflow](http://stackoverflow.com/questions/tagged/firefox-addon), or the [developer mailing list](https://mail.mozilla.org/listinfo/dev-addons). You can also talk with us on [Matrix](https://wiki.mozilla.org/Matrix) in the [Add-ons room](https://mzl.la/2u8ZGbg) (for general development questions) or in the [add-on reviewers room](https://bit.ly/32Odp4i) (for questions about an add-on review).

File: community.md [hunk 2]
```diff
@@ -63,7 +63,9 @@ Subscribe to our [mailing list](https://mail.mozilla.org/listinfo/dev-addons) de
 
 ### Questions about add-on development
 
-You can get help and support by posting to [Mozilla's community forum](https://discourse.mozilla.org/c/add-ons), [Stack Overflow](http://stackoverflow.com/questions/tagged/firefox-addon), or the [developer mailing list](https://mail.mozilla.org/listinfo/dev-addons).
+You can get help and support by posting to [Mozilla's community forum](https://discourse.mozilla.org/c/add-ons), [Matrix Chat Rooms](https://wiki.mozilla.org/Matrix): [#add-ons](https://bit.ly/2PKCHuH), [#addons-reviewers](https://bit.ly/32Odp4i), [Stack Overflow](http://stackoverflow.com/questions/tagged/firefox-addon), or the [developer mailing list](https://mail.mozilla.org/listinfo/dev-addons).
+
```
Comment (caitmuenster) [inline community.md]: Let's delete this.

File: community.md [hunk 3]
```diff
@@ -63,7 +63,9 @@ Subscribe to our [mailing list](https://mail.mozilla.org/listinfo/dev-addons) de
...
+If you already use Matrix, then connect to *#addon-reviewers:mozilla.org*. If you have a Firefox account you can use [chat.mozilla.org/#/room/#addon-reviewers:mozilla.org](https://mzl.la/2x5jNs1) .
```
Comment (caitmuenster) [inline community.md]: Don't forget to update the "last updated" and "updated by" metadata on the page! 😉

Screen Shot 2020-03-03 at 12 40 45 PM

**Comment by muffinresearch** [comment]
@caitmuenster this probably won't build by itself, when it's ready to land let me know and I'll re-push to check the build passes.

**Comment by grlwholifts** [comment]
@caitmuenster @championshuttler I'm not sure why the links keep getting shortened to bit.ly, whereas in my local machine, they look original. Should I use web editor for this?

**Comment by championshuttler** [comment]
No we need to use shortened links only as longer links are not working with Jekyll

File: community.md [hunk 4]
```diff
@@ -4,9 +4,9 @@ title: Community
 permalink: /community/
 published: true
 tags: [community, contribute, WebExtensions]
-contributors: [caitmuenster]
+contributors: [grlwholifts]
```
Comment (championshuttler) [inline community.md]:
```suggestion
contributors: [caitmuenster, grlwholifts]
```

**Pull request merged**